SFconv/sfReader.h

```diff
@@ -9,9 +9,9 @@
 #define INLINE_START	2
 #define INLINE_END		3
 #define INLINE_MARKER	4
-#define UNI_REPLACEMENT_CHAR    (uint32_t)0x0000FFFD
-#define UNI_MAX_BMP             (uint32_t)0x0000FFFF
-#define UNI_MAX_UTF16           (uint32_t)0x0010FFFF
+#define UNI_REPLACEMENT_CHAR    static_cast<uint32_t>(0x0000FFFD)
```

Comment:
I searched around for commentary on what seems to be called a “function-style cast” in C++. I don't have a C++ book handy. There seem to be conflicting accounts of whether it is better than a `static_cast`. (Here's a [StackOverflow discussion](https://stackoverflow.com/q/4474933).) Some people say that `type(value)` is exactly the same as `(type)value`. Scott Meyers [says](http://www.aristeia.com/EC3E/3E_item27.pdf):

> There is no difference in meaning between these forms; it’s purely a matter of where you put the parentheses. I call these two forms _old-style casts_.

At another place, he says:

> Somehow, deliberate object creation doesn’t “feel” like a cast, so I’d probably use the function-style cast instead of the static_cast in this case. Then again, code that leads to a core dump usually feels pretty reasonable when you write it, so perhaps you’d best ignore feelings and use new-style casts all the time.

Based on what I've read, I would lean toward using `static_cast`, but I'm happy to defer to your experience and judgment. Done at d7d9e47e6658ec847a2f3983f3e813fbb43891b7.